### DISCUSSION TITLE
QRZ confirmations

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#856] [Ideas / Feature Requests]
If I'm not mistaken, it looks like QRZ.com's API can return whether a QSO has been confirmed by the other party or not. It would be great to have that data integrated into Cloudlog alongside LoTW and eQSL.

**Comment by antonov124** [comment]
Is it possible to do an extra Confirmed Field for Qrz and other services. Where i can manually add a Confirmation?
Otherwise ive to choose qsl received for "confirmed" 
Happy New year

**Reply by phl0** [reply]
This does not make sense IMHO as the ADIF standard does not define fields for qrz.com confirmations.

**Reply by int2001** [reply]
it'll be ONE-WAY. I'm currently preparing the "QSL" of QRZ.com here: https://github.com/magicbug/Cloudlog/pull/2792
